api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -72,18 +75,42 @@ public Trace() {
     }
 
     /**
-     * @return the id
+     * @return the traceId
+     */
+    public String getTraceId() {
+        return traceId;
+    }
+
+    /**
+     * This method sets the trace id. When the fragment and trace id
```

Comment:
Probably would be good to have on fields aswell. We don't currently publish javadocs but may do in the future.